src/tools/asm2wasm.cpp

```diff
@@ -78,22 +78,39 @@ int main(int argc, const char *argv[]) {
            [](Options *o, const std::string &) {
              std::cerr << "--no-opts is deprecated (use -O0, etc.)\n";
            })
-      .add("--emit-potential-traps", "-i", "Emit instructions that might trap, like div/rem of 0", Options::Arguments::Zero,
+      .add("--emit-potential-traps", "-i",
```

Comment:
oh, is this keeping the old flags for compatibility?

Reply:
Yeah, should I not? And if so, should I remove --imprecise as well?
Actually if we keep it I should update --imprecise to say "old name for --trap-mode=allow" now, to keep that consistent.

Reply:
I think we should consider `asm2wasm` as an implementation detail of emscripten, and aggressively eliminate legacy compatibility things.

Reply:
Yeah, I think these are internal implementation details, just between emcc and asm2wasm. So we can remove old stuff.

But we will need to land this and the emscripten PRs at the same time, and make sure the emscripten one updates the binaryen tag to a new tag on binaryen after that merge, so it uses the new code immediately.